### PR TITLE
Feat/add code comments

### PR DESCRIPTION
**Pull request by kriskuiper**
## Description
This PR adds code comments where needed, also renamed `get-events.js` to `get-events-data.js` because that naming sounds more reasonable. Also put every data-related helper to `src/lib` instead of `lib` because it is more src-code bound then for instance the `render` function which we use in our serverless functions.

### CODE REVIEW

**Comment by deannabosschert** [comment]
Zal ik even wachten met deze mergen tot die van Gijs erdoorheen is?

File: src/lib/format-component-data.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return content.map(contentBlock => {
+    return {
+      // Return everything contentBlock was before and add the two properties
+      // below.
+      ...contentBlock,
+
+      // Every component also needs a component slug for the modular-content
+      // to figure out which component it has to load.
+
+      // Input: Test component
```
[the remark this answers is not in the document]
Comment (kriskuiper) [reply src/lib/format-component-data.js:17]: Que?

[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/lib/format-component-data.js:17]: Like, the input is the name that is given to the component in Storyblok.
So; with capital letter.

File: src/lib/format-component-data.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      // Output: test-component
+
+      // test-component/test-component.html is the template to use.
+      componentSlug: slugify(contentBlock.component, { lower: true }),
+
+      // We have to give every component events so we can use that inside the
```
[the remark this answers is not in the document]
Comment (deannabosschert) [reply src/lib/format-component-data.js:23]: True.

**Comment by gijslaarman** [comment]
Yeeted

**Pull request closed without merging**